src/App.tsx

```diff
@@ -1,7 +1,34 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type Props = {};
+
+type State = {
+  string: string
+};
+
+export class App extends React.Component<Props, State> {
+  state = {
+    string: '',
```

Comment:
it's not a good idea to store a whole string here, it's better to move the string to jsx and only store a value of pressedKey in the state